jscomp/build_tests/uncurried_accessors/input.js

```diff
@@ -0,0 +1,15 @@
+var p = require("child_process");
```

Comment:
I think a build test is not required for this, as the issue shows with some simple files that don't compile.
Adding a file to `jscomp/test` should be enough, as compile errors would be picked up.

Reply:
Plus the compiled output would be checked in.

Reply:
Ok great thanks for the feedback. I'll do that 👍🏼